### PR TITLE
Add edit button to `PresentOffer` component

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add button to open price calculator after showing the price.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1138** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1138" target="_blank">Graphite</a> 
    * **PR #1140** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1140" target="_blank">Graphite</a> 
      * **PR #1141** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1141" target="_blank">Graphite</a> 
        * **PR #1144** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1144" target="_blank">Graphite</a> 
          * **PR #1146** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1146" target="_blank">Graphite</a> 
            * **PR #1151** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1151" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1151?utm_source=stack-comment).

File: apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx [hunk 1]
```diff
@@ -93,6 +90,11 @@ export const OfferPresenter = ({
             <Text as="p" align="center" size="xxl">
               {displayPrice}
             </Text>
+            <button style={{ width: '100%' }} onClick={onClickEdit}>
```
Comment (alebedev) [inline apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx:93]: To be made styled component later?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx]: Sometimes I just miss tailwind 😅

**Pull request merged**